pymc3/step_methods/arraystep.py

```diff
@@ -63,6 +64,29 @@ def step(self, point):
         apoint = self.astep(bij.map(point), *inputs)
         return bij.rmap(apoint)
 
+class FastArrayStep(BlockedStep):
```

Comment:
I think we mostly won't use it, but it it slightly easier to use ArrayStep since there isn't that business with shared variables. However the main reason I've left it in is I didn't want to change all of the samplers yet.